Samples/NetworkMediaStreamer/RTPMediaStreamer/build/RTPMediaStreamer.vcxproj

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
+      <Configuration>Debug</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|Win32">
+      <Configuration>Release</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">

```

Comment:
>Release|x64" [](start = 35, length = 12)

Are ARM/ARM64 unsupported?

Reply:
good point. I need to test it on arm first. I shall add that once I have completed the testing.

---
In reply to: [491067114](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491067114) [](ancestors = 491067114)

Reply:
I checked with ROS team, the ARM is currently not used for ROS on Windows. So this is not a priority for ROS. I can take the ARM support as a separate PR. Ideally it should just work. But testing is required before i enable that.

---
In reply to: [491084948](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491084948) [](ancestors = 491084948,491067114)

Reply:
[https://microsoft.visualstudio.com/_workitems/edit/29536275?fullScreen=True](https://microsoft.visualstudio.com/_workitems/edit/29536275?fullScreen=True)

---
In reply to: [492399433](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r492399433) [](ancestors = 492399433,491084948,491067114)